utils/src/lib.rs

```diff
@@ -179,21 +228,6 @@ mod tests {
         assert_eq!(result.len(), result.capacity());
     }
 
-    #[test]
```

Comment:
Can you re-add this test?

```
    #[test]
    fn test_union_unique_zero_length() {
        let namespace = b"";
        let msg = b"message";

        let length_encoding = vec![0];
        let mut expected = Vec::with_capacity(length_encoding.len() + namespace.len() + msg.len());
        expected.extend_from_slice(&length_encoding);
        expected.extend_from_slice(msg);

        let result = union_unique(namespace, msg);
        assert_eq!(result, expected);
        assert_eq!(result.len(), result.capacity());
    }
    ```
    
  (I think you removed the modified one rather than reverting to the previous one)

Reply:
oops, restored.